src/partners/coral/index.js

```diff
@@ -0,0 +1,619 @@
+/* eslint-disable */
```

Comment:
small change, can you move this file to ` src/dapps/SecureTransaction/` we like to keep all related files in one location.
also remove `/* eslint-disable */`